projects/demo/src/modules/components/file-uploader/examples/1/index.less

```diff
@@ -0,0 +1,34 @@
+@import '@taiga-ui/core/styles/taiga-ui-local.less';
+
+:host {
+    @media not @tui-mobile {
+        display: flex;
+    }
+}
+
+tui-file {
+    //noinspection CssInvalidPropertyValue
+    inline-size: -webkit-fill-available;
```

Comment:
It's "inline-size: stretch", autoprefixer would apply proper rules for our browserslist

Reply:
fixed